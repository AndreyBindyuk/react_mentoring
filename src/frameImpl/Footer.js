import React from 'react';
import '../css/App.css';

class Footer extends React.Component{
    render(){
        return(
            <div className='footer'>
            <span>Copyright 2019 Company name, Designed by Andrey. All rights reserved</span></div>
        );
    }
}

export default Footer;