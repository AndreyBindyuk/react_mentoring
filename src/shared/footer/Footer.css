.footer {
  height: 100px;
  width: 100%;
  background-color: rgb(187, 240, 247);
}
.footer span {
  margin: 20px;
}
