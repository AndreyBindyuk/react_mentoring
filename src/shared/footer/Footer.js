import React from 'react';
import './Footer.css';

class Footer extends React.Component{
    render(){
        return(
            <div className='footer'>
            <span>Copyright 2019 Company name, Designed by Andrey. All rights reserved</span></div>
        );
    }
}

export default Footer;