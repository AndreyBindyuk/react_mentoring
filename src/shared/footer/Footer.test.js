import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('<Footer />', () => {
  it('render internal components correctly', () => {
    const ModalTriggerComponent = shallow(<Footer />);
    expect(ModalTriggerComponent).toMatchSnapshot();
  });
});
