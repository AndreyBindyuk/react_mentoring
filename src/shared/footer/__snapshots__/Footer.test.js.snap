// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render internal components correctly 1`] = `ShallowWrapper {}`;
